index.html

```diff
@@ -500,33 +500,36 @@
     }
     </pre>
 
-    ##### Taking a picture with Red Eye Reduction supported and used
+    ##### Update device zoom
     <pre class='example'>
-    navigator.mediaDevices.getUserMedia({video: true}).then(gotMedia, failedToGetMedia);
+    var imageCapture;
+
+    navigator.mediaDevices.getUserMedia({video: true}).then(gotMedia).catch(failedToGetMedia);
```

Comment:
The changes here lgtm.  I was wondering if we could upload this examples in some kind of codepen/jsfiddle or similar, and link (informatively) from these examples, so the casual spec reader can just click through and see these wonderful demos in action :smile: -- my question would be if w3c has some specific way of doing it.